.github/workflows/alpha.yaml

```diff
@@ -0,0 +1,406 @@
+            --dbname ship-cloud --driver postgres
+
+      - name: Upload schema artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: schema
+          path: ./migrations/fixtures/schema/fixtures.sql
+
+
+  generate-fixtures:
+    runs-on: ubuntu-18.04
+    needs: [generate-schema]
+    container:
+      image: replicated/gitops-builder:buildkite
+      options: --user root
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Download schema artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: schema
+        path: ./migrations/fixtures/schema
+
+    - name: Build fixtures
+      run: make -C migrations/fixtures deps build run
+
+    - name: Upload fixtures artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: fixtures
+        path: ./migrations/fixtures/fixtures.sql
+
+
+  publish-fixtures:
+    runs-on: ubuntu-18.04
+    needs: [generate-fixtures]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - uses: azure/docker-login@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+    - name: Download fixtures artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: fixtures
+        path: ./migrations/fixtures
+
+    - name: Publish fixtures
+      run: |
+        make -C migrations/fixtures publish
+
+
+  build-schema-migrations:
+    runs-on: ubuntu-18.04
+    needs: [publish-fixtures]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - uses: azure/docker-login@v1
+      env:
+        DOCKER_CONFIG: ./migrations/.docker
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+    - name: Get tags
+      id: get_tag
+      uses: ./actions/version-tag
+
+    - name: Build
+      env:
+        DOCKER_CONFIG: ./.docker
+      run: |
+        make -C migrations schema-alpha
+
+
+  test_web:
+    runs-on: ubuntu-18.04
+    needs: [build-schema-migrations]
+    container:
+      image: replicated/gitops-builder:buildkite
+      options: --user root
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Node.js environment
+      uses: actions/setup-node@v2
+      with:
+        node-version: '10'
+
+    - name: Install cross-env
+      run: npm i -g cross-env
+      shell: bash
+
+    # - name: Test web
+    #   env:
+    #     PACT_BROKER_USERNAME: ${{ secrets.PACT_BROKER_USERNAME }}
+    #     PACT_BROKER_PASSWORD: ${{ secrets.PACT_BROKER_PASSWORD }}
+    #   run: |
+    #     make -C kotsadm/web deps test publish-pact
+
+      # TODO: migrate retry logic for pacts
+      # retry:
+      #   automatic:
+      #     # this command exiting with status 2 typically means that the yarn package failed to download
+      #     - exit_status: 2
+      #       limit: 5
+
+
+  build_web:
```

Comment:
This  needs to run in tagged workflow as well

Reply:
it does

Reply:
I didn't see `build-kotsadm` anywhere else